src/ppx/grfx/grfx_pipeline.cpp

```diff
@@ -153,7 +169,10 @@ void FillOutGraphicsPipelineCreateInfo(
         pDstCreateInfo->colorBlendState.blendAttachmentCount = pSrcCreateInfo->outputState.renderTargetCount;
         for (uint32_t i = 0; i < pDstCreateInfo->colorBlendState.blendAttachmentCount; ++i) {
             switch (pSrcCreateInfo->blendModes[i]) {
-                default: break;
+                default: {
+                    pDstCreateInfo->colorBlendState.blendAttachments[i].colorWriteMask = grfx::ColorComponentFlags::RGBA();
```

Comment:
this line should be out of the switch so that for all blend mode we set the color mask to RGBA?

Reply:
in that case you can probably remove it? we should assert if the blendmode is undefined